config/config.exs

```diff
@@ -2,7 +2,7 @@ import Config
 
 config :screen_checker,
   solari_screen_list_module: ScreenChecker.SolariScreenList,
-  gds_dms_username: "mbtadata@gmail.com"
+  gds_dms_username: "screens-hardware@mbta.com"
```

Comment:
I wonder if this should be an environment variable (not a secret), so bots won't collect it from this public repository. If we're going to need some infra changes anyway...

Reply:
Good suggestion, will do.